popups/compare_commit_view.html

```diff
@@ -0,0 +1,23 @@
+<html>
+<style>
+{css}
+</style>
+<body>
+<div class="header-wrapper"><h2>Keyboard Shortcuts</h2></div>
+
+<h3>Actions</h3>
+<ul>
+  <li><code><span class="shortcut-key">enter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>show commit options</code></li>
+  <li><code><span class="shortcut-key">m&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>toggle commit info on the bottom</code></li>
+  <li><code><span class="shortcut-key">.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>go to next commit</code></li>
+  <li><code><span class="shortcut-key">,&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>go to previous commit</code></li>
+  <li><code><span class="shortcut-key">f&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>show commits diff</code></li>
+</ul>
+
+<h3>Other</h3>
+<ul>
+  <li><code><span class="shortcut-key">?&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>show this help popup</code></li>
+  <li><code><span class="shortcut-key">r&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>refresh the page</code></li>
+</ul>
+</body>
+</html>
```

Comment:
When hitting the `git: compare against ...` command or `git: log` -> select commit -> `compare commit against ...`, the `commit_commit_view` is shown. And pressing `?` in `commit_commit_view` will show this popup.